projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/Untitled.ipynb

```diff
@@ -0,0 +1,48 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "deefa60e-f66b-4af3-a558-b9ed19d10d39",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "vdk_ipython extension introduces a magic command for Jupyter.\n",
+    "The command enables the user to load VDK for the current notebook.\n",
+    "VDK provides the job_input API, which has methods for:\n",
+    "    * executing queries to an OLAP database;\n",
+    "    * ingesting data into a database;\n",
+    "    * processing data into a database.\n",
+    "See the IJobInput documentation for more details.\n",
```

Comment:
here can you add this link to the documentation for I job input
https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/api/job_input.py